### ISSUE TITLE
WCAG 2.4.1: Page must have means to bypass repeated blocks (html)

### ISSUE THREAD

**Issue by ada-developer** [ada-cat/accessibility#16]
#### Issue
`Page must have means to bypass repeated blocks` ([bypass](https://dequeuniversity.com/rules/axe/3.2/bypass?application=msftAI))
#### Target application
[Accessible University Demo Site - Inaccessible Version](https://markreay.github.io/AU/before.html)
#### Element path
html
#### Snippet
`<html>`
#### How to fix
    Fix any of the following:
      No valid skip link found
      Page does not have a header
      Page does not have a landmark region
#### Environment
Microsoft Edge version 75.0.139.4
This accessibility issue was found using Accessibility Insights for Web 2.3.0 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
